Codeinwp/neve
# fix: infinite scroll with masonry scrolling to top of page

### Summary
Fixes issue with infinite scroll and masonry scrolling to the top of the page.


### Will affect visual aspect of the product

NO

### Test instructions


- Have posts on your site
- Enable masonry and infinite scroll
- The page should not scroll up when new posts are inserted in the masonry grid 


Closes #1467.
